src/components/Results/Results.tsx

```diff
@@ -21,13 +21,15 @@ export const Results = () => {
   useEffect(() => {
     const searchQuery = searchText.trim().length > 0 ? `search=${searchText}&searchBy=title&` : '';
     const sortQuery = `sortBy=${sort}&sortOrder=asc`;
-    const filterQuery = filter === FILTER_OPTIONS[0].value ? '' : `&filter=${filter}`;
+    const filterQuery = filter === FILTER_OPTIONS[0].value ? '' : `&filter=${filter}&`;
     const query = `?${searchQuery}${filterQuery}${sortQuery}`;
     setQueryParams(query);
   }, [searchText, filter, sort]);
 
   const { data, isLoading, isFetching, isError } = useGetMoviesQuery(queryParams);
 
+  console.log(queryParams);
```

Comment:
Please do not commit console.log to you repo

Reply:
thanks, forgot to remove them before commiting